components/menu/menu.js

```diff
@@ -76,13 +76,29 @@ class Menu extends HierarchicalViewMixin(LitElement) {
 	}
 
 	focus() {
-		if (this._items[0].role === 'menuitemradio') {
+		if (this.getMenuType() === 'menu-radio') {
 			this._focusSelected();
 		} else {
 			this._focusFirst();
 		}
 	}
 
+	getMenuType() {
```

Comment:
Might be nice if this returned `'menu'` if there were no items? That way it always returns something.